frontend/webpack.config.ts

```diff
@@ -56,7 +56,6 @@ const config: webpack.Configuration = {
       },
       {
         test: /\.s?css$/,
-        exclude: /node_modules/,
```

Comment:
@priley86 I know you needed this in your tables PR... What impact does this have on our builds?

Reply:
this is required due to react-core [change in import methodology](https://github.com/patternfly/patternfly-react/pull/1592). CSS will be loaded directly from `node_modules/@patternfly` from our JS requires.

I used the quick and easy method in the tables PR, but we should probably be more selective so that directories unexpected are not scanned for CSS. Looks we can do something like:
```
      {
        test: /\.s?css$/,
        exclude: /node_modules\/(?!(@patternfly)\/).*/,
      }
```
or
```
      {
        test: /\.s?css$/,
        exclude: function(modulePath) {
          return /node_modules/.test(modulePath) &&
              !/node_modules\/@patternfly/.test(modulePath);
        },
      }
```

Webpack does not seem to respect:
```
      {
        test: /\.s?css$/,
        exclude: /node_modules/,
        include: [/node_modules\/@patternfly/],
      }
```

source: https://github.com/webpack/webpack/issues/2031#issuecomment-283517150

cc: @dgutride

Reply:
@priley86 It looks like this change pulls in a bunch of CSS we don't use and increases our total CSS by 50%

Reply:
@spadgett  - I'd agree with that assessment and this is a perfect use case for our more advanced PatternFly css architecture.  For OpenShift, my recommendation would be to  investigate use of a null loader (https://webpack.js.org/loaders/null-loader/) for the css require statements that have been introduced by react-core.  Since your team is importing the full patternfly css from @ patternfly/patternfly - component specific css can be discarded.

Reply:
@dgutride we're not importing the full patternfly css... unfortunately null loader just results in a bunch of missing styles :/

Reply:
@spadgett  - just reread your https://github.com/openshift/console/blob/ed60c15f42f0a172b66ad5578871e3604c8484ec/frontend/public/vendor.scss file - I misunderstood how you are pulling it in.  In that case, I'd encourage you to load the css using Patrick's method here.  The component css is pulled in individually by component (and while yes, it will add to your css), please take a look at the size of your DOM as you navigate through the site.  Most of the emotion tags (except for tippy and toolbar) have been removed and bloat over time is removed because the css is now extracted instead of inserted into the head.

Reply:
The end result is that you have the same amount of CSS but it is managed far more efficiently and you don't have to worry about garbage collection between routes or DOM bloat over time (and caching of css files leads to better efficiency).

Reply:
With @priley86's method, we're still seeing the inclusion of CSS for components we're not using.  Is this intended?

Reply:
>  still seeing the inclusion of CSS for components we're not using

It in fact appears to be all [components](https://github.com/patternfly/patternfly-next/tree/master/src/patternfly/components) and [utilities](https://github.com/patternfly/patternfly-next/tree/master/src/patternfly/utilities).  This does not seem desirable.

Reply:
@priley86 confirmed all of PF4 CSS is pulled in only during dev as treeshaking occurs for prod and only relevant CSS is bundled.  

The increase in the dist vendor bundle size is expected as PF4 CSS is no longer being injected in to head via emotion, but instead is being bundled.

@spadgett, this PR is ready for final review.

Reply:
please inspect webpack production build when checking this pf4 upgrade. there may be additional reductions we can add later.